stdlib/2.7/decimal.pyi

```diff
@@ -1,17 +1,25 @@
 # Stubs for decimal (Python 2)
-#
-# NOTE: This dynamically typed stub was automatically generated by stubgen.
 
-from typing import Any, SupportsAbs, SupportsFloat, SupportsInt
+from typing import (
+    Any, Dict, NamedTuple, Optional, Sequence, Tuple, Union,
+    SupportsAbs, SupportsFloat, SupportsInt,
+)
 
-ROUND_DOWN = ...  # type: Any
-ROUND_HALF_UP = ...  # type: Any
-ROUND_HALF_EVEN = ...  # type: Any
-ROUND_CEILING = ...  # type: Any
-ROUND_FLOOR = ...  # type: Any
-ROUND_UP = ...  # type: Any
-ROUND_HALF_DOWN = ...  # type: Any
-ROUND_05UP = ...  # type: Any
+_Decimal = Union[Decimal, int]
```

Comment:
I think modern Decimal is lenient about floats and just accepts them. I recommend just adding float here, else you have to add it to many other places (in our server code I found problems comparing floats to decimals.